lib/content-services/src/lib/upload/components/upload-version-button.component.ts

```diff
@@ -47,6 +51,7 @@ export class UploadVersionButtonComponent extends UploadButtonComponent implemen
     }
 
     ngOnInit() {
+        this.file = this.newFileVersion;
```

Comment:
having 2 properties in this case does not make sense, just use `file`

Reply:
you're right, changed that, my bad.